ux.symfony.com/README.md

```diff
@@ -2,48 +2,63 @@
 
 Source code for [ux.symfony.com](https://ux.symfony.com).
 
-## Contributing
+## Installation
 
-### Local Development
+### Source code
 
-1. Install the project:
-    ```bash
-    git clone git@github.com:symfony/ux
-    cd ux/ux.symfony.com/
-    ```
+Install the project:
+```bash
+git clone git@github.com:symfony/ux
+cd ux/ux.symfony.com/
+```
 
-2. Install the dependencies:
-    ```bash
-    symfony composer install
-    ```
+Install the dependencies:
+```bash
+symfony composer install
+```
 
-3. (optional) Configure Docker:
-    ```bash
-    docker compose up -d
-    ```
+### Services
 
-5. Run database migrations:
-    ```bash
-    symfony console doctrine:migration:migrate
-    ```
+(optional) Configure Docker to launch Mercure
+```bash
+docker compose up -d
+```
 
-6. Populate the database:
-    ```bash
-    symfony console app:load-data
-    ```
+### Database
 
-7. Compile the Sass files:
-    ```bash
-    symfony console sass:build --watch
-    ```
+Run database migrations:
+```bash
+symfony console doctrine:migration:migrate
+```
+
+Populate the database:
+```bash
+symfony console app:load-data
+```
 
-8. Start the web server:
-    ```bash
-    symfony server:start -d
-    ```
+### Assets
 
+Download the importmap packages locally:
+```bash
+symfony console importmap:install
+```
+
+Compile the Sass files:
+```bash
+symfony console sass:build
+
+# (optional) Add the --watch flag to automatically recompile the Sass files on change.
+symfony console sass:build --watch
+```
+
+### Local server
+
+Start the local web server (in background):
+```bash
+symfony server:start -d
```

Comment:
Since Symfony CLI 5.7.0 we are able to auto start the docker compose services when starting server with the file `.symfony.local.yaml` and the key `workers.docker_compose: ~`.
Doc : https://symfony.com/doc/current/setup/symfony_server.html#configuring-workers

Reply:
Not sure to understand… you’d like it to be mentionned ?

Reply:
The `symfony server:start` command is able to run the `docker compose up -d` at the same time, this will remove the service step

Reply:
Ok i understand.. but that would need to cmmit the .symfony.local.yaml no ? As Docker is only required for Mercure and that concern only a part of the site, i think i'd rather keep it optional

Reply:
This file can also run the sass watch/build step: https://symfony.com/bundles/TailwindBundle/current/index.html#symfony-cli (tailwind-bundle docs but same applies to sass-bundle)

Reply:
What is your vision on the ".local" thing. I install many binaries locally to "not depend on" Docker images, but i'm not sure to represent most of users.

Maybe suggest symfony-cli team to handle .symfony.dev.yaml Vs .symfony.local.yaml 

(hmm.. it may already be possible i'll check)

Reply:
I'm not sure here as the (optional) docker stuff adds a bit of a wrench in the idea.

Reply:
My vision was more to reduce the number of steps to launch the project, if we can do it in one command, we should to it.

What do you mean by `handle .symfony.dev.yaml vs .symfony.local.yaml` ? To me the dev env is the integration with platform.sh : https://docs.platform.sh/guides/symfony/local.html which also supported with the symfony-cli

Reply:
II think we should not commit the .symfony.local.yaml, as it's a ".local" file, dedicated to machine settings, and that could be annoying to edit this file everytime i want to work on the website, because IP port is hardcoded in it, or because it requires Docker :)

Reply:
It seems "nothing", but there is a worker, and i don't find its logs

Reply:
You think "Docker" but if the sass watch tries to run natively in background that's not good either

(here, it's a special case i don't care that much.. but as this file is the one saving the proxy TLD, the local host... it should not be encouraged to share/commit it imho)

Reply:
But i 100% agree with you that it would be ideal to just run "composer install" + "symfony serve" and have a working environment !

Reply:
So let's do how you prefer... initially i just wanted to add "importmap:install" hahaha

Reply:
Why don't we try adding `.symfony.local.yaml` and see if we have anyone with issues. @smnandre, you could test w/o docker and @maxhelias, you could test with.

I'm thinking the `.symfony.local.yaml` file would look like the following:

```yaml
workers:
    docker_compose: ~
    sass:
        cmd: ['symfony', 'console', 'sass:build', '--watch']
```

Reply:
Well while you do so, i'm gonna work a script to ignore this one and use mine, while not overriding this one 😅

Reply:
(and this is not to say "don't do it", i think it's a good idea... not just for me :p Let's see if i'm the only one :)

Reply:
> ```yaml
> workers:
>     docker_compose: ~
>     sass:
>         cmd: ['symfony', 'console', 'sass:build', '--watch']
> ```

You can commit in this PR ? Or i can update it with this test if you want ?

Reply:
> i'm gonna work a script to ignore this one and use mine

Well as soon as one person wants to ignore, it is probably going to be a problem. Perhaps this is a good reason to not add it.

Reply:
No seriously, Let's try. 
And I need to decorate symfony cli in all manners :)

Reply:
> Or i can update it with this test if you want ?

Ok, would you mind adding and testing then?

Reply:
Good